services/agent_manager.py

```
@@ -0,0 +1,57 @@
+async def load_data_to_vector_db(tenant_id: str, conteudo_loja: str, products_df: pd.DataFrame):
+    logger.info(f"Carregando dados para o VectorDBManager do tenant: {tenant_id}")
+    db = SessionLocal()
+    try:
+        vector_db_manager = VectorDBManager(collection_name=tenant_id)
+
+        # Adicionar o conteúdo do arquivo .txt
+        documents_to_add = [conteudo_loja]
+        metadatas_to_add = [{"tenant_id": tenant_id, "type": "company_info"}]
+
+        # Adicionar os produtos do Excel com formatação aprimorada
+        for index, row in products_df.iterrows():
```

Ruff (0.12.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)